ext/Errno/Errno_pm.PL

```diff
@@ -469,7 +473,6 @@ under the same terms as Perl itself.
 
 =cut
 
-# ex: set ro:
 ESQ
```

Comment:
I will accept your point about adding a comment (now squashed into 3rd commit), but otherwise I'm going to stick with the approach I've taken.  If you disagree, then I recommend you file your own p.r. and we'll let the PSC choose.

Reply:
I second @bram-perl's point.There are any number of ways you can obfuscate the modeline. I suspect a simple escape on the colon would do it actually, which would be even easier than brams proposal.  A crafty way would be to replace "ex" with `${\'ex'}` :-)

FWIW, @jkeenan I think this is a pretty reasonable request that is too minor to justify escalation to the PSC. There are a handful of simple ways you can obfuscate the modeline so a modeline parser doesn't treat it as /the/ modeline for the file and I find it hard to understand why you object. If it were my patch id be grateful for the suggestion.